### PR TITLE
Create dataset_to_xarray_notebook.ipynb

### PR DESCRIPTION
**Pull request by ThomasMGeo**
In your pull request, please provide the following information:

* What datasets does your viewer handle?

Forecast Model Data  &  Forecast Products and Analyses come to mind. Any dataset that works well in siphon and is gridded x, y, (z,t) types of data should work. 

* A summary of what your viewer does.

Ingests data via siphon and creates an xarray dataset and data array.

Hopefully putting pr https://github.com/Unidata/tds/pull/318 in the correct spot.

Thanks!
-TM

### CODE REVIEW

File: threddsDev/notebooks/dataset_to_xarray_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,1108 @@
...
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.13"
+  },
+  "viewer_info": {
+   "accepts": {},
+   "description": "The TDS default viewer attempts to plot any Variable contained in the Dataset."
```
Comment (haileyajohnson) [inline threddsDev/notebooks/dataset_to_xarray_notebook.ipynb:1098]: don't think this is right, haha

File: threddsDev/notebooks/dataset_to_xarray_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,1108 @@
...
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.13"
+  },
+  "viewer_info": {
+   "accepts": {},
```
Comment (haileyajohnson) [inline threddsDev/notebooks/dataset_to_xarray_notebook.ipynb:1097]: I'll take a look at what needs to go in here to target the right datasets

**Review by haileyajohnson — CHANGES_REQUESTED** [review]

File: threddsDev/notebooks/dataset_to_xarray_notebook.ipynb (hunk 2 above)
Comment (ThomasMGeo) [reply threddsDev/notebooks/dataset_to_xarray_notebook.ipynb:1097]: Thanks!

**Comment by haileyajohnson** [comment]
Ok, last couple things:
1) We want this on the main thredds server, so it actually needs to go under thredds/notebooks/ (you'll need to create that directory)
2) It looks like you've checked the notebook in with output, which is fine if you want it that way, but wanted to double check :)
3) The "accepts" metadata parameter should work if it looks like this:
```  
"viewer_info": {
   "accepts": {
    "accept_catalogs": ["Unidata THREDDS Data Server - NCEP models"]
   },
   "description": "Your description here"
  }
```

... I think. We'll start with that and I'll troubleshoot it if that doesn't show up they way we want.

**Comment by ThomasMGeo** [comment]
OK, 1 & 2 should be fixed (I think!)

Unsure exactly where 3 goes, in the notebook? 

Thanks!
-TM

File: thredds/notebooks/dataset_to_xarray_notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.13"
+  },
```
Comment (haileyajohnson) [inline thredds/notebooks/dataset_to_xarray_notebook.ipynb:169]: This is where 3 goes, in the raw metadata :)

**Comment by ThomasMGeo** [comment]
Thanks! Appreciate it. :)

**Review by haileyajohnson — APPROVED** [review]

**Comment by haileyajohnson** [comment]
🎉

**Pull request merged**